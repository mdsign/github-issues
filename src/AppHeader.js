import React from 'react';

const AppHeader = () => {
    return(
        <header className="l-app-header">
          <h3> Github Issues - JavaScript Frameworks</h3>
        </header>
    );    
}

export default AppHeader;