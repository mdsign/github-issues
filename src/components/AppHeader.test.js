import React from 'react';
import AppHeader from './AppHeader.js';
import renderer from 'react-test-renderer';

it('renders header correctly', () => {
  const tree = renderer
    .create(<AppHeader />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
