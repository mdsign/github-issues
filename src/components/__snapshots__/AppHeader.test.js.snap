// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header correctly 1`] = `
<header
  className="l-app-header"
>
  <h3>
     Github Issues - JavaScript Frameworks
  </h3>
</header>
`;
